reflex/constants/event.py

```diff
@@ -47,6 +48,7 @@ class SocketEvent(SimpleNamespace):
 
     PING = "ping"
     EVENT = "event"
+    HEALTH = "health"
```

Comment:
not needed with the socket event removed

Reply:
Fixed it